front/src/app/groups/[id]/page.tsx

```diff
@@ -45,64 +329,504 @@ const GroupViewPage = () => {
+                              width: "100%",
+                              height: "260px",
+                              marginTop: "2px",
+                              marginBottom: "0px",
+                              overflowY: "auto",
+                              border: "none",
+                              boxShadow: "none",
+                            }}
+                          >
+                            <FormGroup>
+                              {options
+                                .filter((option) =>
+                                  option.name
+                                    .toLowerCase()
+                                    .includes(searchTerm.toLowerCase())
+                                )
+                                .map((option) => (
+                                  <FormControlLabel
+                                    key={option.id}
+                                    control={
+                                      <Checkbox
+                                        checked={selectedCheckboxes.some(
+                                          (checkbox) =>
+                                            checkbox.id === option.id
+                                        )}
+                                        onChange={() =>
+                                          handleCheckboxChange(option.id)
+                                        }
+                                        style={{
+                                          color: "#265073",
+                                          marginBottom: "0px",
+                                          marginTop: "0px",
+                                        }}
+                                      />
+                                    }
+                                    label={
+                                      <span
+                                        style={{
+                                          maxWidth: "300px",
+                                          overflow: "hidden",
+                                          display: "inline-block",
+                                          whiteSpace: "unset",
+                                          textOverflow: "ellipsis",
+                                          wordBreak: "break-all",
+                                          marginTop: "0px",
+                                          marginBottom: "0px",
+                                          paddingTop: "3px",
+                                          paddingBottom: "0px",
+                                        }}
+                                      >
+                                        {option.name}
+                                      </span>
+                                    }
+                                  />
+                                ))}
+                              {options.filter((option) =>
+                                option.name
```

Comment:
Handle "TypeError: Cannot read properties of null" on runtime

Reply:
TypeError has been resolved